main.go

```diff
@@ -100,22 +106,22 @@ func main() {
 		os.Exit(1)
 	}
 
-	tokenPath := "/workspace/quaytoken"
-	tokenContent, err := ioutil.ReadFile(tokenPath)
-	if err != nil {
-		setupLog.Error(err, "unable to read quay token")
+	getReadConfigFunc := func(path string) func() string {
+		return func() string {
+			tokenContent, err := ioutil.ReadFile(path)
+			if err != nil {
+				setupLog.Error(err, fmt.Sprintf("unable to read %s", path))
+			}
+			return strings.TrimSpace(string(tokenContent))
+		}
 	}
-	orgPath := "/workspace/organization"
-	orgContent, err := ioutil.ReadFile(orgPath)
-	if err != nil {
-		setupLog.Error(err, "unable to read quay organization")
-	}
-	quayClient := quay.NewQuayClient(&http.Client{Transport: &http.Transport{}}, strings.TrimSpace(string(tokenContent)), "https://quay.io/api/v1")
+
+	quayClient := quay.NewQuayClient(&http.Client{Transport: &http.Transport{}}, getReadConfigFunc(tokenPath), "https://quay.io/api/v1")
 	if err = (&controllers.ComponentReconciler{
 		Client:           mgr.GetClient(),
 		Scheme:           mgr.GetScheme(),
 		QuayClient:       quayClient,
-		QuayOrganization: strings.TrimSpace(string(orgContent)),
+		QuayOrganization: getReadConfigFunc(orgPath),
```

Comment:
I think the organization should not be changed. We have to have a constant here.

Reply:
I mean, the code is confusing.